### PR TITLE
Update copyq_ko.ts

### PR DESCRIPTION
**Pull request by VenusGirl**
Update Korean

### CODE REVIEW

File: translations/copyq_ko.ts [hunk 1]
```diff
@@ -6,12 +6,12 @@
     <message>
         <location filename="../src/ui/aboutdialog.ui" line="14"/>
         <source>About</source>
-        <translation>정보</translation>
+        <translation>정보 - 한국어 번역: 비너스걸💋</translation>
```
Comment (hluk) [inline translations/copyq_ko.ts]: Would it be OK to add yourself to the about dialog instead?

https://github.com/hluk/CopyQ/blob/af7a2a53aafaa5fe690c470cedecad166bfcd08d/src/gui/aboutdialog.cpp#L111-L131

**Review by hluk — COMMENTED** [review]
Thanks for your hard work! :pray: 

I suggest using [Weblate](https://hosted.weblate.org/engage/copyq/?utm_source=widget) to translate this app next time. It is usually more convenient and safer.

I will merge this, I just have one comment.

**Comment by VenusGirl** [comment]
I got it!

File: translations/copyq_ko.ts (hunk 1 above)
Comment (VenusGirl) [reply translations/copyq_ko.ts]: OK!
"VenusGirl", "venusgirl@outlook.com"

Comment (hluk) [reply translations/copyq_ko.ts]: Sorry, this is not yet resolved. I prefer having simpler menu items, not containing any contacts (BTW, there are also other people working on translations).

But feel free to add your contact in the aboutdialog.cpp (ideally mentioning that you do the translation).

**Review by VenusGirl — COMMENTED** [review]
It has been modified.

**Comment by hluk** [comment]
Thank you! 🙏

**Pull request merged**